src/libcmd/repl.cc

```diff
@@ -833,9 +820,18 @@ void NixRepl::evalString(std::string s, Value & v)
 }
 
 
+void NixRepl::runNix(Path program, const Strings & args, const std::optional<std::string> & input)
+{
+    if (runNixPtr)
+        (*runNixPtr)(program, args, input);
+    else
+        throw Error("Cannot run '%s', no method of calling the Nix CLI provided", program);
```

Comment:
```suggestion
        throw Error("Cannot run '%s' because no method of calling the Nix CLI was provided. This is a configuration problem pertaining to how this program was built. See Nix 2.25 release notes.", program);
```

- pet peeve: unidiomatic use of comma
- let's give some context

Ideally we could be more specific and give a URL.

Reply:
you could also use a semicolon to keep the brevity